src/excellentexport.ts

```diff
@@ -150,7 +150,17 @@ const ExcellentExport = function() {
             // Create sheet
             workbook.SheetNames.push(name);
             const worksheet = XLSX.utils.aoa_to_sheet(dataArray, {sheet: name} as XLSX.AOA2SheetOpts);
-            
+            for (var cellName in worksheet) {
+                if (utils.hasContent(worksheet[cellName])) {
+                    var cell = worksheet[cellName];
+                    if (utils.hasContent(cell.v)) {
+                        if(cell.v[0] == '=') {
+                            cell.f = cell.v.toString().substring(1)

```

Comment:
If the "=" is removed, it will not get into the final Excel file. Then opening will not get it as a formula.
Did this work for you?

Reply:
Yes it works but only for XLSX format
Probably for XLS needed few modifications
I've tried to launch index.html in Chrome and Edge
XLSX example: 
![image](https://github.com/jmaister/excellentexport/assets/7942871/c9646641-d291-42fb-93ba-e71da0d7d33f)
in XLS format - it really doesn't work (and cells remains empty)
![image](https://github.com/jmaister/excellentexport/assets/7942871/fa79ccad-5626-4c0d-a724-d0fdafd08dd8)

Reply:
Looks like it is not supported to write formula in XLS format at all :(
https://docs.sheetjs.com/docs/csf/features/formulae/